src/components/flatSelect/examples/FlatSelectExamples.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { IFlatSelectOptionProps } from '../FlatSelectOption';
+import { UUID } from '../../../utils/UUID';
+import { FlatSelectConnected } from '../FlatSelectConnected';
+import { ISvgProps, Svg } from '../../svg/Svg';
+import { IContentProps } from '../../content/Content';
+
+export class FlatSelectExamples extends React.Component<any, void> {
+
+  render() {
+    const defaultFlatSelectOption: IFlatSelectOptionProps[] = [
+      {
+        option: { content: 'Option 1' },
+      }, {
+        option: { content: 'Option 2' },
+      }, {
+        option: { content: 'Option 3' },
+      },
+    ];
+
+    const svg: ISvgProps = {
+      svgName: 'domain-google',
+      svgClass: 'icon',
+    };
+    const getPrepend: IContentProps = { content: () => <Svg {...svg} />, classes: ['mr1'] };
+    const getAppend: IContentProps = { content: () => <Svg {...svg} />, classes: ['ml1'] };
```

Comment:
I think I already said this in another review but those two variables should be `const prepend` and `const append` since they are not getters